htdocs/js/wui.js

```diff
@@ -149,6 +149,13 @@ vz.wui.addEntity = function(entity) {
 		entity.eachChild(function(child) {
 			child.loadTotalConsumption();
 		}, true); // recursive
+	
+	// create push subscriptions
+	entity.subscribe();	
+	entity.eachChild(function(child) {
+		child.subscribe();
+	}, true); // recursive
+	
```

Comment:
Immer oder nur falls active? Evtl. den active check in `subscribe` mit einbauen?